DotNet.DocsTools/GitHubCommunications/GitHubClient.cs

```diff
@@ -1,47 +1,32 @@
-﻿using Polly.Contrib.WaitAndRetry;
-using Polly;
-using Polly.Retry;
-using System;
+﻿using System.Net.Http.Headers;
+using System.Net.Mime;
+using System.Text.Json;
 
 namespace DotnetDocsTools.GitHubCommunications;
 
-public sealed class GitHubClient : IGitHubClient, IDisposable
+internal class GitHubClient : IGitHubClient
```

Comment:
This entire file should be reverted. The changes that these overwrite are undesirable, and remove the transient fault handling that we paired on adding in #92.

Reply:
I made the version with Polly the shared version.